.github/workflows/integration.yml

```diff
@@ -88,16 +88,17 @@ jobs:
       timeout-minutes: 10
       run: |
         chmod +x ./build/fleetctl
-        ./build/fleetctl preview --no-hosts
+        ./build/fleetctl preview --no-hosts --disable-open-browser
         ./build/fleetctl config set --address ${{ needs.gen.outputs.address }}
         ./build/fleetctl get enroll-secret
         docker compose -f ~/.fleet/preview/docker-compose.yml logs --follow fleet01 fleet02 &
         # Wait for all of the hosts to be enrolled
         EXPECTED=3
-        until [ $(./build/fleetctl get hosts --json | wc -l | tee hostcount) -ge $EXPECTED ]; do
+        until [ $(./build/fleetctl get hosts --json | grep -v "No hosts found" | wc -l | tee hostcount) -ge $EXPECTED ]; do
```

Comment:
I still need `"No hosts found"` because the previous step downloads fleetctl from `main`.